pkg/test/resourcefixture/testdata/basic/speech/v1alpha1/speechphraseset/speechphraseset-minimal/_http.log

```diff
@@ -5,15 +5,12 @@ X-Goog-Request-Params: name=projects%2F${projectId}%2Flocations%2Fglobal%2Fphras
 
 404 Not Found
 Content-Type: application/json; charset=UTF-8
-Grpc-Status-Details-Bin: CAUSW1VuYWJsZSB0byBmaW5kIFBocmFzZVNldCBzcGVlY2hwaHJhc2VzZXQtbWluaW1hbC16ZHVqeTV0dmFubXJ0eGkgZnJvbSBwcm9qZWN0IDYwMDEzOTU2NjExNi4a+QEKJHR5cGUuZ29vZ2xlYXBpcy5jb20vdXRpbC5TdGF0dXNQcm90bxLQAQgFEgdnZW5lcmljGltVbmFibGUgdG8gZmluZCBQaHJhc2VTZXQgc3BlZWNocGhyYXNlc2V0LW1pbmltYWwtemR1ank1dHZhbm1ydHhpIGZyb20gcHJvamVjdCA2MDAxMzk1NjYxMTYuKmYLEIbWjScaXRJbVW5hYmxlIHRvIGZpbmQgUGhyYXNlU2V0IHNwZWVjaHBocmFzZXNldC1taW5pbWFsLXpkdWp5NXR2YW5tcnR4aSBmcm9tIHByb2plY3QgNjAwMTM5NTY2MTE2Lgw=
```

Comment:
Non-blocker: Maybe we should clear these headers/fields when recording the golden logs?